packages/preview/modern-cv/0.8.0/lib.typ

```diff
@@ -0,0 +1,851 @@
+    weight: "light",
+    fill: color-darknight,
+  )
+  set block(
+    above: 0.75em,
+    below: 1.25em,
+  )
+  set par(leading: 0.65em)
+  block(above: 0.5em)[
+    #body
+  ]
+}
+
+/// The base item for resume entries. This formats the item for the resume entries. Typically your body would be a bullet list of items. Could be your responsibilities at a company or your academic achievements in an educational background section.
+/// - title (string): The title of the resume entry
+/// - location (string): The location of the resume entry
+/// - date (string): The date of the resume entry, this can be a range (e.g. "Jan 2020 - Dec 2020")
+/// - description (content): The body of the resume entry
+/// - title-link (string): The link to use for the title (can be none)
+/// - accent-color (color): Override the accent color of the resume-entry
+/// - location-color (color): Override the default color of the "location" for a resume entry.
+#let resume-entry(
+  title: none,
+  location: "",
+  date: "",
+  description: "",
+  title-link: none,
+  accent-color: default-accent-color,
+  location-color: default-location-color,
+) = {
+  let title-content
+  if type(title-link) == "string" {
+    title-content = link(title-link)[#title]
+  } else {
+    title-content = title
+  }
+  block(above: 1em, below: 0.65em)[
+    #pad[
+      #justified-header(title-content, location)
+      #if description != "" or date != "" [
+        #secondary-justified-header(description, date)
+      ]
+    ]
+  ]
+}
+
+/// Show cumulative GPA.
+/// *Example:*
+/// #example(`resume.resume-gpa("3.5", "4.0")`)
+#let resume-gpa(numerator, denominator) = {
+  set text(
+    size: 12pt,
+    style: "italic",
+    weight: "light",
+  )
+  text[Cumulative GPA: #box[#strong[#numerator] / #denominator]]
+}
+
+/// Show a certification in the resume.
+/// *Example:*
+/// #example(`resume.resume-certification("AWS Certified Solutions Architect - Associate", "Jan 2020")`)
+/// - certification (content): The certification
+/// - date (content): The date the certification was achieved
+#let resume-certification(certification, date) = {
+  justified-header(certification, date)
+}
+
+/// Show a list of skills in the resume under a given category.
+/// - category (string): The category of the skills
+/// - items (list): The list of skills. This can be a list of strings but you can also emphasize certain skills by using the `strong` function.
+#let resume-skill-item(category, items) = {
+  set block(below: 0.65em)
+  set pad(top: 2pt)
+  
+  pad[
+    #grid(
+      columns: (20fr, 80fr),
+      gutter: 10pt,
+      align(right)[
+        #set text(hyphenate: false)
+        == #category
+      ],
+      align(left)[
+        #set text(
+          size: 11pt,
+          style: "normal",
+          weight: "light",
+        )
+        #items.join(", ")
+      ],
+    )
+  ]
+}
+
+/// ---- End of Resume Template ----
+
+/// ---- Coverletter ----
+
+#let default-closing(lang_data) = {
```

Comment:
To be more consistent I would rename `lang_data` to `lang-data`.

Reply:
Thanks, done!